samples/track-run.js

```diff
@@ -13,6 +13,8 @@ function handle (req, res) {
 
 function startBench () {
   const instance = autocannon({
+    connections: 100,
+    duration: 2,
```

Comment:
why this change?

Reply:
Leftover from my testing, haven't yet reviewed my PR.